e2e/cypress/integration/menus/status_dropdown_spec.js

```diff
@@ -36,6 +48,106 @@ describe('Status dropdown menu', () => {
         cy.get('.MenuWrapper.status-dropdown-menu .Menu__content.dropdown-menu').should('be.visible');
     });
 
+    it('MM-T671 Prompting to set status to online from Away', () => {
+        cy.apiLogin(user1);
+        cy.visit(testChannelUrl);
+
+        // # Use the status drop-down on your profile pic to go Away
+        cy.get('img.Avatar').click();
+        cy.findByText('Away').click();
+
+        // * Your status stays offline in your view
+        cy.get('.away--icon').should('be.visible');
+
+        // # Log out
+        cy.apiLogout();
+
+        // # Log back in
+        cy.apiLogin(user1);
+        cy.visit(testChannelUrl);
+
+        // # On modal that asks if you want to be set Online, select Yes.
+        cy.get('.modal-content').within(() => {
+            cy.findByText('Your Status is Set to "Away"').should('be.visible');
+            cy.findByText('Yes, set my status to "Online"').click();
+        });
+
+        // * Your status is online in your view
+        cy.get('.online--icon').should('be.visible');
+
```

Comment:
@saturninoabril, if I place a cy.pause() at this point, I can view see who user1 is and open a DM with that user as a test user in MM. After some time, the user status shows them as online. However, after I login through the cypress test below this line and open a DM with user1, user1 never shows up as online. Thus, this test fails after this point. I'm not sure how to simulate this in cypress. I tried adding a minute of waite time after opening the DM, but user1 never shows up as online

Reply:
I'm seeing that too. I tried several approach and it's not working either. Will take a look deeper after the Hackathon.  Thanks for checking!

Reply:
I got it working by adding extra check to status after click. Please try below and let me know if will work on you as well.
```javascript
cy.get('.modal-content').within(() => {
    cy.findByText('Your Status is Set to "Away"').should('be.visible');
    cy.findByText('Yes, set my status to "Online"').click();

    cy.apiGetUserStatus(user1.id).its('status.status').should('equal', 'online');
});

// * Your status is online in your view
cy.get('.online--icon').should('be.visible');
```